### PR TITLE
Fix: Remove deprecated `descr` field

### PR DESCRIPTION
**Pull request by Septias**
Update webxdc types and remove usage of deprecated `descr` field

### CODE REVIEW

**Review by adbenitez — APPROVED** [review]

**Comment by adbenitez** [comment]
wondering why CI didn't seem to have run for this PR, maybe this needs a rebase?

**Comment by Septias** [comment]
I want to merge this into main directly after add_nix was merged

**Review by adbenitez — APPROVED** [review]
works like a charm! `npm run check` fails locally, prettier needs to be applied and we should wait for CI to run and pass

**Pull request merged**
